### PR TITLE
Support backends OpenStack Stein

### PR DESCRIPTION
**Pull request by h00130372**
Support acceptance test against OpenStack Stein release in OpenLab

Add periodic jobs for ansible/packer/docker-machine/manageiq
against OpenStack Stein release.

Related: theopenlab/openlab#231

### CODE REVIEW

**Comment by kiwik** [comment]
Hi @h00130372 , you are trying to do this work https://github.com/theopenlab/openlab/issues/231 , right? If it is, please add link into commit message and PR describe to make others clear.

File: zuul.d/jobs.yaml [hunk 1]
```diff
@@ -454,6 +463,20 @@
     vars:
       os_sdk: openstacksdk
 
+- job:
+    name: openstacksdk-ansible-functional-devstack-stein
+    parent: init-test
+    description: |
+      Run openstacksdk ansible functional tests against a stein devstack using git stable-2.7 branch version of ansible.
```
Comment (liu-sheng) [inline zuul.d/jobs.yaml]: please put this description into two lines.

Comment (h00130372) [reply zuul.d/jobs.yaml]: I corrected this last time when I found the description only show the first line content on the job web page(http://status.openlabtesting.org/jobs), like the base job https://github.com/theopenlab/openlab-zuul-jobs/blob/master/zuul.d/jobs.yaml#L7, please check this, or I can open an issue to trace this bug.

Comment (liu-sheng) [reply zuul.d/jobs.yaml]: we can land this firstly, maybe there is a fix in zuul in future

**Pull request merged**